spec/models/schedule_rule_spec.rb

```diff
@@ -357,7 +357,7 @@
       assert @rule2.valid?
 
       # find past tuesday, and build calendar objects
-      @tuesday    = ScheduleRule.sunday_last + 2.days
+      @tuesday = Time.current.beginning_of_week(:sunday).to_date + 2
```

Comment:
If we're no longer using `.day` in this line, shouldn't we take it out in the line below as well?